test-data/unit/fixtures/property.pyi

```diff
@@ -16,6 +16,7 @@ class classmethod: pass
 class list: pass
 class dict: pass
 class int: pass
+class float: pass
```

Comment:
I had to add this because the `typing-medium.pyi` fixture references `float`, so the two fixtures wouldn't work together without this.